### DISCUSSION TITLE
What comes next in this sequence?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1163] [Galaxy Trivia] [answered]
```
1 4 10 22 46 ?
```

**Comment by CelioAmaral** [comment, marked as answer]
The logical rule of the sequence is: multiply the previous term by 2 and add this result with 2 units.

Numerical Sequence
A numerical sequence is a list made up of numbers that has a generally well-defined order. Furthermore, it contains what we know as the law of formation, or law of recurrence, which allows us to find the next terms in the sequence.

It is possible to observe that its logical rule is given by multiplying the previous term by 2 and adding this result to 2.

1 x 2 = 2 + 2 = 4

4 x 2 = 8 + 2 = 10

10 x 2 = 20 + 2 = 22

22 x 2 = 44 + 2 = 46

46 x 2 = 92 + 2 = 94

94.

According to the question, the numerical sequence is 1, 4, 10, 22, 46, 94.
